CONTRIBUTING.md

```diff
@@ -44,7 +70,36 @@ You are an awesome project now! Feel encouraged to tell others about it by addin
 [![Mentioned in Awesome Go](https://awesome.re/mentioned-badge.svg)](https://github.com/avelino/awesome-go)  
 [![Mentioned in Awesome Go](https://awesome.re/mentioned-badge-flat.svg)](https://github.com/avelino/awesome-go)
 ```
+## Maintenance expectations for projects listed here
+
+To prevent removal from go-awesome, your project must maintain the following quality standards:
+- On-going development while maintaining code quality. Official release should be at least once a year if the project is ongoing. OR, 
```

Comment:
It is very likely that the owners/maintainers of a project haven't added it on awesome-go but someone else had.
I believe in such cases, they will be dealt with on a case-by-case basis?